src/Game.cpp

```diff
@@ -179,6 +179,15 @@ bool Game::ProcessEvents(const SDL_Event& event)
 		{
 			return false;
 		}
+		else if (event.window.event == SDL_WINDOWEVENT_SIZE_CHANGED)
+		{
+			int width, height;
```

Comment:
I know we do it in other places but for readability, don't declare two variables on the same line.
If you really want one line try this:
```
glm::ivec2 resolution;
_window->GetSize(resolution,x, resolution.y);
```